stdlib/public/SwiftShims/LibcShims.h

```diff
@@ -121,14 +124,23 @@ static inline __swift_size_t _swift_stdlib_malloc_size(const void *ptr) {
   return malloc_usable_size(CONST_CAST(void *, ptr));
 }
 #elif defined(_WIN32)
+#define HAS_MALLOC_SIZE 1
 static inline __swift_size_t _swift_stdlib_malloc_size(const void *ptr) {
   extern __swift_size_t _msize(void *ptr);
   return _msize(CONST_CAST(void *, ptr));
 }
 #else
-#error No malloc_size analog known for this platform/libc.
+#define HAS_MALLOC_SIZE 0
+
+static inline __swift_size_t _swift_stdlib_malloc_size(const void *ptr) {
+  return 0;
+}
 #endif
 
+static inline int _swift_stdlib_has_malloc_size() {
+  return HAS_MALLOC_SIZE;
+}
```

Comment:
Changed to `__swift_bool`. 

Any particular reason for the `!= 0`?

Reply:
> Any particular reason for the `!= 0`?

No reason. I think the implicit conversion is good for `__swift_bool` (aka C++ `bool` or C99 `_Bool`). But it would be bad for Objective-C [`BOOL`](https://developer.apple.com/documentation/objectivec/bool?language=objc#discussion) (aka `signed char` on macOS).

Reply:
OK, added it.